tcks/microprofile-health/pom.xml

```diff
@@ -0,0 +1,66 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-build-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../../build-parent/pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-tck-microprofile-health</artifactId>
+    <name>Quarkus - TCK - MicroProfile Health</name>
+
+    <properties>
+        <maven.deploy.skip>true</maven.deploy.skip>
+        <!-- TODO move the version to the build parent -->
+        <microprofile-health-api.version>1.0</microprofile-health-api.version>
```

Comment:
Same here, better do it now than having a TODO?

I wonder if it should be in the BOM near the implementation version so we think about upgrading them together.